### PR TITLE
ISSUE_TEMPLATE added

### LINKED ISSUE #6: Add issue template(s)
If there would be an Issue and PR template that can really help maintainers in order to review the changes.

### PR DESCRIPTION
**Pull request by abh139**
I have added the Issue Template, Please add them.
This PR will close #6

### CODE REVIEW

**Review by verma-kunal — APPROVED** [review]
LGTM 👍

**Comment by verma-kunal** [comment]
Nice work @abh139

**Comment by anubha-v-ardhan** [comment]
> I have added the Issue Template, Please add them. This PR will close #14

I think this PR will close #6 and not #14

File: .github/Issue_Template/Doc.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of issue
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
```
Comment (AvineshTripathi) [inline .github/Issue_Template/Doc.yml:14]: Is there any reason we have screenshot in the doc template, if so can you elaborate more

**Review by AvineshTripathi — COMMENTED** [review]
Also I see some similar content in all the three files and that leads to a question to folks if we really require all the three or any one would work 
cc @anubha-v-ardhan @verma-kunal

**Comment by abh139** [comment]
> > I have added the Issue Template, Please add them. This PR will close #14
> 
> I think this PR will close #6 and not #14

Yes Edited.

File: .github/Issue_Template/Doc.yml (hunk 1 above)
Comment (abh139) [reply .github/Issue_Template/Doc.yml:14]: In the Doc section If someone wants some suggestions in updating the pic depicting any flow of a process or any improvement or wanted to add some pictures then they can add, also there Is also a word written there "if applicable" then only they have to add.

**Comment by abh139** [comment]
@AvineshTripathi have a look now, Is there any further changes required.

**Comment by AvineshTripathi** [comment]
LGTM!!

thank you for the contribution

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by AvineshTripathi** [comment]
cc @anubha-v-ardhan @verma-kunal

File: .github/Issue_Template/Feature.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: 💡Feature 
+title: '[FEATURE] <description>'
+labels: ['⭐ goal: addition','triage/accepted','triage/awaiting']
```
Comment (anubha-v-ardhan) [inline .github/Issue_Template/Feature.yml:3]:
```suggestion
labels: ['triage/awaiting', 'enhancement']
```
We don't have `goal: addition` label. Maintainers replace `triage/awaiting` with `triage/accepted` after going through the issue. I propose GitHub's default `enhancement` label for Feature requests.

File: .github/Issue_Template/Other.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
```
Comment (anubha-v-ardhan) [inline .github/Issue_Template/Other.yml:2]:
```suggestion
description: File any other issues.
```
Black issues cannot be created due to `required: true` value. I think we don't need to state that.

File: .github/Issue_Template/Other.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+title: '[OTHER]'
+labels: ['🚦 status: awaiting triage','triage/accepted']
```
Comment (anubha-v-ardhan) [inline .github/Issue_Template/Other.yml:4]:
```suggestion
labels: ['triage/awaiting']
```
We don't have a `status: awaiting triage` label. Also we need to give `triage/awaiting` when issue is created. Maintainers will replace it with `triage/accepted` if the issue is accepted.

File: .github/Issue_Template/bug.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+name:  Bug
+description: Report an issue to help improve the project.
```
Comment (anubha-v-ardhan) [inline .github/Issue_Template/bug.yml:2]:
```suggestion
name:  Bug Report
description: Report a bug encountered with Kubesimplify Website
title: '[Bug] <description>'
```

File: .github/Issue_Template/Other.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
+title: '[OTHER]'
```
Comment (anubha-v-ardhan) [inline .github/Issue_Template/Other.yml:3]:
```suggestion
title: '[OTHER] <description>'
```

File: .github/Issue_Template/bug.yml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+labels: ['🛠 goal: fix','triage/awaiting','triage/accepted']
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the issue.
```
Comment (anubha-v-ardhan) [inline .github/Issue_Template/bug.yml:9]:
```suggestion
      description: A brief description of the bug.
```

File: .github/Issue_Template/bug.yml [hunk 8]
```diff
@@ -0,0 +1,26 @@
+name:  Bug
+description: Report an issue to help improve the project.
+labels: ['🛠 goal: fix','triage/awaiting','triage/accepted']
```
Comment (anubha-v-ardhan) [inline .github/Issue_Template/bug.yml:3]:
```suggestion
labels: ['triage/awaiting','bug']
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
In addition to suggestions below, please also change the file locations to `.github/ISSUE_TEMPLATE/*.yml`. GitHub only recognizes files inside `ISSUE_TEMPLATE` as issue templates :)

**Comment by anubha-v-ardhan** [comment]
We'll also need a template for Notes, as they'll be part of the website as well but I think we'll add it in future when notes part starts taking shape.

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!
Thank you @abh139 :)

**Pull request merged**